leaf-ui/js/objects.js

```diff
@@ -221,7 +222,30 @@ class AnalysisConfiguration {
 	 * Add a new AnalysisResult to the analysisResults array.
 	 * @param {AnalysisResult} analysisResult 
 	 */
-	addAnalysis(analysisResult) {
+	addResult(analysisResult) {
 		this.analysisResults.push(analysisResult);
 	}
+
+	/**
+	 * Updates Config Values from AnalysisRequest
+	 */
+	updateAnalysis(analysisRequest){
```

Comment:
I feel like the word "analysis" in this class implies the `AnalysisResult`, but I'm not sure what exactly is the best method name. For now I'll just suggest "update"
```suggestion
	update(analysisRequest){
```

Reply:
I feel like update is too vague (could suggest id as well which could cause confusion once we implement naming, doesn't say what you are updating explicitly, etc) but hear your concern. Thoughts on updateAnalysisValues ?

Reply:
how about updateAnalysisConfig?